src/Illuminate/Database/Console/Seeds/WithoutModelEvents.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Illuminate\Database\Console\Seeds;
+
+use Illuminate\Database\Eloquent\Model;
+
+trait WithoutModelEvents
+{
+    /**
+     * Prevent model event handles from being executed by the given callback.
```

Comment:
Could maybe be better written as this?

```suggestion
     * Prevent model events from being dispatched by the given callback.
```